openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -97,6 +98,7 @@ class Meta(object):
     course_video_url = TextField(null=True)
     effort = TextField(null=True)
     self_paced = BooleanField(default=False)
+    instructor_info = JSONField(null=False, blank=False)
```

Comment:
@douglashall, or we can change this. I was not satisfied with this structure. It should be a list of dicts. like this
`[
{instructor1 data},
{instructor2 data}
]`
Let me know if you want to change this.